docs/ug/report-config.yaml

```diff
@@ -0,0 +1,11 @@
+repoUrl: github.com/user/repo
+reportTitle: RepoSense Report
+authorDisplayName: Sample Author
+authorGithubId: Sample Author Github ID
+branches:
+  - name: Branch 1
+    blurb: Blurb 1
```

Comment:
Blurb should allow a few paragraph of markdown text. So, yml might not be the right format for this config file.

Reply:
@damithc There are ways to incorporate multi-line strings into YAML files [here](https://yaml-multiline.info/), we could perhaps include this into the docs to let users know how they can tailor the blurbs to suit their use cases (multi-paragraph vs long single paragraph text)